pkg/aws/aws_client/role.go

```diff
@@ -496,12 +523,12 @@ func (client *AWSClient) UpdateAssumeRolePolicy(roleName string, assumeRolePolic
 }
 
 func (client *AWSClient) UpdateAssumeRolePolicyForSharedVPCRole(roleName string, installerRoleArn string,
```

Comment:
What will happen if `controlPlaneOperatorRoleArn` set to empty? We have to consider about classic shared vpc cluster role creation.

Reply:
@OAharoni-RedHat Yes please. For better maintenance, let's comment with explanation.

Reply:
Yes. It looks better. Thank you.